src/inversion_scripts/operators/operator_utilities.py

```diff
@@ -171,34 +182,17 @@ def concat_tracers(run_id, gc_date, config, sv_elems, n_elements, baserun=False)
                                     
 
     """
-    prefix = config['OutputPath'] + '/' + config['RunName'] + '/jacobian_runs'
-    j_dir = os.path.expandvars(f"{prefix}/{config['RunName']}_{run_id}/OutputDir")
+    prefix = os.path.expandvars(
+        config['OutputPath'] + '/' + config['RunName'] + '/jacobian_runs'
+    )
+    j_dir = f"{prefix}/{config['RunName']}_{run_id}/OutputDir"
     file_stub = gc_date.strftime('GEOSChem.SpeciesConc.%Y%m%d_0000z.nc4')
     dsmf = xr.open_dataset(
         '/'.join([j_dir,file_stub]),
         chunks = 'auto'
     )
     keepvars = [f'SpeciesConcVV_CH4_{i:04}' for i in sv_elems]
 
```

Comment:
As mentioned above, this PR includes changing BC and OH jacobian rundirs to include an additional tracer now named for the current state vector element. This was done for consistency with how the emission perturbations are handled and to avoid the need for different treatment of tracer names throughout IMI scripts.